fastlane/Fastfile

```diff
@@ -23,7 +23,10 @@ platform :android do
       sh("git checkout -b #{branch_name}")
       git_commit(path: "*/libraries.json", message: "Update libraries.json with up-to-date libraries to scan.")
       git_commit(path: "*/build.gradle", message: "Update versionCode and versionName")
+      sh("git push origin #{branch_name}")
```

Comment:
Been trying to create a pull request for a branch that didn't exist on Github. 🤕